templates/edit_profile.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% block title %}Edit profile data{% endblock %}
+{% block content %}
+    <!--{% from "macros.html" import input with context %}-->
+    {% from "macros.html" import field_input with context %}
+    <div class="cover-container d-flex h-80 p-3 mx-auto flex-column">
+        <main role="main" class="inner cover">
+            <div class="container my_container">
+                <div class="row">
+                    <div class="col-md-12">
+                        <div class="wrap">
+                            <button type="button" style="white" class="close" data-dismiss="modal" aria-label="Close" onclick="window.location.href='{{ url_for('library.index')}}';">
```

Comment:
delete `style="white"` it doesn't do anything ;)

Reply:
yep, thanks!